### DISCUSSION TITLE
Latest beta versions.

### DISCUSSION THREAD

**Discussion by tzmosh** [Stellarium/stellarium#2657] [General]
My computer, windows 8.1, does not allow installation of latest beta versions.
I get a message that the software is not compatible with the operating system.

**Comment by gzotti** [comment]
Qt6-based versions of Stellarium (these are all recent betas that are future-oriented) require Windows 10 or later. We will still issue releases (but not all betas) for outdated (W8.*) or obsolete (W7) versions.

**Comment by alex-w** [comment]
> My computer, windows 8.1, does not allow installation of latest beta versions.
> I get a message that the software is not compatible with the operating system.

Oops... It was mistake in description of requirements for Win64 beta - thanks for note!
